scripts/code-check

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+yarn tsc:check
```

Comment:
Circular dependency. `yarn` calls `sh` calls `yarn`.

Reply:
It's not really a "circular" dependency. Yarn run can call other yarn runs. I did however make this call the other scripts directly